src/gui/transferlistfilterswidget.cpp

```diff
@@ -420,8 +423,11 @@ void TrackerFiltersList::handleFavicoDownload(const QString& url, const QString&
 void TrackerFiltersList::handleFavicoFailure(const QString& url, const QString& error)
 {
     Q_UNUSED(error)
-    if (url.endsWith(".ico", Qt::CaseInsensitive))
+    if (url.endsWith(".ico", Qt::CaseInsensitive)) {
         downloadFavicon(url.left(url.size() - 4) + ".png");
+    } else {
+        downloadFavicon("https://www.google.com/s2/favicons?domain=" + getHost(url));
```

Comment:
Can't understand how that is possible. Could you explain, please? I see that he loop downloadFavicon()... handleFavicoFailure() will run forever.